orchagent/orch.cpp

```diff
@@ -729,7 +741,7 @@ task_process_status Orch::handleSaiCreateStatus(sai_api_t api, sai_status_t stat
                     SWSS_LOG_WARN("SAI_STATUS_SUCCESS is not expected in handleSaiCreateStatus");
                     return task_success;
                 default:
-                    SWSS_LOG_ERROR("Encountered failure in create operation, exiting orchagent, SAI API: %s, status: %s",
```

Comment:
@donNewtonAlpha should we keep the original log message? Since it does exit OA

Reply:
+1. There should be no change on this.

Reply:
Fixed